magprime/automated_emails.py

```diff
@@ -19,69 +20,94 @@ def __init__(self, event):
-           lambda a: a.staffing and a.has_shifts_in(c.CHIPSPACE) and a.trusted_in(c.CHIPSPACE), sender='MAGFest ChipSpace <chipspace@magfest.org>')
+           lambda a: a.staffing and a.has_shifts_in(c.CHIPSPACE) and a.trusted_in(c.CHIPSPACE),
+           ident='magprime_chipspace_trusted_volunteer',
+           sender='MAGFest ChipSpace <chipspace@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest Chipspace', 'chipspace_untrusted.txt',
-           lambda a: a.staffing and a.has_shifts_in(c.CHIPSPACE) and not a.trusted_in(c.CHIPSPACE), sender='MAGFest ChipSpace <chipspace@magfest.org>')
+           lambda a: a.staffing and a.has_shifts_in(c.CHIPSPACE) and not a.trusted_in(c.CHIPSPACE),
+           ident='magprime_chipspace_untrusted_volunteer',
+           sender='MAGFest ChipSpace <chipspace@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest Staff Suite rules', 'food_volunteers.txt',
-           lambda a: a.staffing and a.has_shifts_in(c.FOOD_PREP) and not a.trusted_in(c.FOOD_PREP), sender='MAGFest Staff Suite <chefs@magfest.org>')
+           lambda a: a.staffing and a.has_shifts_in(c.FOOD_PREP) and not a.trusted_in(c.FOOD_PREP),
+           ident='magprime_food_untrusted_volunteer',
+           sender='MAGFest Staff Suite <chefs@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest message from Chef', 'food_trusted_staffers.txt',
-           lambda a: a.staffing and a.has_shifts_in(c.FOOD_PREP) and a.trusted_in(c.FOOD_PREP), sender='MAGFest Staff Suite <chefs@magfest.org>')
+           lambda a: a.staffing and a.has_shifts_in(c.FOOD_PREP) and a.trusted_in(c.FOOD_PREP),
+           ident='magprime_food_trusted_volunteer',
+           sender='MAGFest Staff Suite <chefs@magfest.org>')
 
 AutomatedEmail(Attendee, 'Want to help run MAGFest poker tournaments?', 'poker.txt',
-               lambda a: a.has_shifts_in(c.TABLETOP), sender='MAGFest Tabletop <tabletop@magfest.org>')
+               lambda a: a.has_shifts_in(c.TABLETOP),
+               ident='magprime_tabletop_volunteer_poker_inquiry',
+               sender='MAGFest Tabletop <tabletop@magfest.org>')
 
 StopsEmail('MAGFest Staff Support', 'staff_support.txt',
-           lambda a: a.assigned_to(c.STAFF_SUPPORT) and not a.trusted_in(c.STAFF_SUPPORT))
+           lambda a: a.assigned_to(c.STAFF_SUPPORT) and not a.trusted_in(c.STAFF_SUPPORT),
+           ident='magprime_staff_support_volunteer',)
 
 # Left here in case the decision to remove is re-visited later. 20160809 RAE
 # MarketplaceEmail('Your MAGFest Marketplace Application has been waitlisted', 'marketplace_auto_waitlisted.txt',
 #                 lambda g: g.status == c.WAITLISTED and g.registered >= c.DEALER_REG_DEADLINE)
 
 StopsEmail('MAGFest Dept Checklist Introduction', 'dept_checklist_intro.txt',
-           lambda a: a.is_single_dept_head and a.admin_account)
+           lambda a: a.is_single_dept_head and a.admin_account,
+           ident='magprime_magprime_dept_checklist_intro')
```

Comment:
You wrote `magprime` twice here. Not super critical but it'd be nice if it was only in the ident once.